doc/method/method.md

```diff
@@ -6,173 +6,348 @@ The assembly process works, in outline, as follows:
  1. Start with an ordered list of source taxonomies S1, S2, ...
  1. Initialize the 'union' taxonomy U to be empty
  1. For each source S:
-     1. The nodes of S are aligned with, or separated from, nodes of U
-        where possible (in a manner explained below)
-     1. Unaligned subtrees of S - i.e. subtrees of S whose tips are tips
-        of S, and that contain no aligned nodes other than the root - are grafted onto U
-     1. Where S provides a more resolved classification than U, unaligned
-        internal nodes of S are 'inserted' into U
- 1. Ad hoc postprocessing steps
- 1. Assign identifiers to the nodes of U
+     1. Load, normalize, and patch S
+     1. Align S to U, i.e. match the nodes of S to nodes of U, where possible
+     1. Merge S into U
+         1. Unaligned subtrees of S (subtrees of S that contain
+            no matched nodes other than their root) are grafted onto U
+         1. Where S provides a more resolved classification than U, 
+            'insert' unmatched nodes of S into U
+ 1. Apply patches and perform ad hoc postprocessing steps
+ 1. Assign OTT identifiers to the nodes of U, by aligning the previous 
+    version of OTT to U
 
 The hierarchical relationships are therefore determined by priority:
 ancestor/descendant relationships in an earlier source S may be
-subdivided by a later source S', but are never overridden.
+refined by a later source S', but are never overridden.
 
 For each new version of OTT, construction begins de novo, so that we
-always get the latest version of the source
-taxonomies.
+always get the latest version of the source taxonomies.  The only
+state that persists from one version of OTT to the next is OTT
+identifier assignment.
 
 Details of each step follow.
 
-## Source taxonomy preparation
+## Source taxonomy import and normalization
 
-Each source taxonomy has its own ingest procedure, usually a file
+Each source taxonomy has its own import procedure, usually a file
 transfer followed by application of a format conversion script.
 E.g. the GBIF taxonomy is downloaded from the GBIF web site, and then
 converted to the Open Tree 'interim taxonomy format' by a python
-script.
-
-Following ingest the following two normalizations are performed:
-
- 1. "Containers" - nodes in the tree that don't represent taxa - are
-    removed and replaced by flags (node annotations indicating edge
-    type); the most prominent being the pseudo-taxon "Incertae sedis"
+script.  Given the ITF files, the taxonomy can be 'loaded' by the
+assembly procedure.
+
+In two cases an imported source taxonomy is split into parts before
```

Comment:
Want a statement explaining why these two cases exist, i.e. feedback from taxonomic experts in this clade. Probably want a more general statement in introduction about the role of expert curation in the system.